packages/react-components/src/organisms/__tests__/GroupCard.test.tsx

```diff
@@ -21,6 +22,13 @@ it('renders the group name linking to the group', () => {
   );
 });
 
+it('renders the state tag for a inactive group', () => {
```

Comment:
please test for the absense of inactive in this test with a rerender

Reply:
On it